.github/workflows/main.yml

```diff
@@ -94,6 +94,15 @@ jobs:
       - name: Set up Pebble
         run: curl --request POST --data '{"ip":"10.30.50.1"}' http://localhost:8055/set-default-ipv4
 
+      - name: update apt package list
+        run: apt update
+
+      - name: install apache2
```

Comment:
This should probably happen in a docker-container..
Start nginx or apache with a custom configuration and mount/copy the hash-file into the container.

Reply:
i hoped it would work this way. very naive :D

Reply:
using a service now 66265e6